packages/@okta/vuepress-site/docs/guides/set-up-event-hook/verify-your-endpoint/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Verify Your Endpoint
+---
+
+After registering the event hook, you need to trigger a one-time verification process, by making a `POST` request to `https://{yourOktaDomain}/api/v1/eventHooks/${id}/lifecycle/verify`. The `id` value here is the system-generated unique ID value for the Event Hook object, which Okta returned in the `eventHook` object when you registered the event hook.
```

Comment:
Event Hook x 2

And I'd shorten this to just say:

"The `id` value here belongs to the Event Hook object which Okta returned when..."

Reply:
Made suggested edit.